README.md

```diff
@@ -1719,7 +1722,7 @@ regarding stock awards and stock options.
   For example, the value of the stock may not increase.
   And if you leave the company before you vest, you *don’t* get back the taxes you’ve
   already paid.
-- ❗ You must file the 83(b) election yourself with the IRS
+- ❗ An employee must file the 83(b) election themselves with the IRS
```

Comment:
Not always an employee. Probably easiest to revert. I think the warning is clearest first person (and is not a definition).

Reply:
Fixed. Agreed.